asapdiscovery-modeling/asapdiscovery/modeling/scripts/make_foldalysis.py

```diff
@@ -0,0 +1,93 @@
+"""
+Build a Fragalysis-style directory structure from the outputs of ColabFold.
+Inside the given output directory, two additional directories will be created:
+aligned and extra_files. The extra_files directory will contain the
+Mpro_compound_tracker_csv.csv file, which can be used as input to the
+prep_proteins.py script in asapdiscoveyr.docking. The aligned directory will
+contain the PDB structure files in the same format as in the Fragalysis
+database.
+"""
+import argparse
+from glob import glob
+import os
+import re
+
+################################################################################
+def get_args():
+    parser = argparse.ArgumentParser(description="")
+
+    parser.add_argument(
+        "-id",
+        "--in_dir",
+        required=True,
+        help="Input directory for this script (output directory from ColabFold.)",
+    )
+    parser.add_argument(
+        "-ic",
+        "--in_csv",
+        required=True,
+        help="CSV file that was used as input for ColabFold.",
+    )
+    parser.add_argument(
+        "-o", "--out_dir", required=True, help="Output directory."
+    )
+
+    return parser.parse_args()
+
+
+def main():
+    args = get_args()
+
+    # Get all the sequence names that were generated by ColabFold
+    # Get rid of the first one because it's just the header line
+    all_ids = [l.split(",")[0] for l in open(args.in_csv).readlines()][1:]
+    print(f"Found {len(all_ids)} sequences", flush=True)
```

Comment:
here raise an error if none are found?

Reply:
done in [cee0e84](https://github.com/choderalab/asapdiscovery/pull/141/commits/cee0e84fe164b4a80e0d86ccf07ef332bc8890a6)